openverse_catalog/dags/common/requester.py

```diff
@@ -59,8 +62,11 @@ def get(self, url, params=None, **kwargs):
         """
         self._delay_processing()
         self._last_request = time.time()
+        request_kwargs = kwargs or dict()
```

Comment:
The bracket declaration is generally faster than function calls in python, so we should try to prefer using that where we can!
```python
[ins] In [1]: %timeit a = {}
14.6 ns ± 0.0481 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

[ins] In [2]: %timeit a = dict()
55.5 ns ± 2.05 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
Wow! Awesome, totally will do. Am I right in thinking that `{}` will always be a `dict`, but if I want a `set`, I should do `set()`?

Reply:
`set` unfortunately doesn't have similar empty object literal, so we do have to use `set()` for that.